code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -22,11 +24,16 @@
 /obj/machinery/particle_accelerator/control_box/New()
 	wires = new /datum/wires/particle_accelerator/control_box(src)
 	connected_parts = list()
+	radio = new(src)

```

Comment:
This HAS to be in Initialize if it is an atom. Otherwise it won't get spawned correctly